modules/auxiliary/admin/http/allegro_rompager_auth_bypass.rb

```diff
@@ -25,8 +25,8 @@ def initialize(info = {})
               ],
               'References' => [
                   ['CVE', '2014-9222'],
-                  ['URL', 'http://mis.fortunecook.ie'],
-                  ['URL', 'http://mis.fortunecook.ie/misfortune-cookie-suspected-vulnerable.pdf'], # list of likely vulnerable devices
+                  ['URL', 'https://web.archive.org/web/20191006135858/http://mis.fortunecook.ie/'],
+                  ['URL', 'https://web.archive.org/web/20190207102911/http://mis.fortunecook.ie/misfortune-cookie-suspected-vulnerable.pd'], # list of likely vulnerable devices
```

Comment:
Note sure what happened here. The updated URL with `pd` file extension is invalid. The URL should be:

```suggestion
                  ['URL', 'https://web.archive.org/web/20190207102911/http://mis.fortunecook.ie/misfortune-cookie-suspected-vulnerable.pdf'], # list of likely vulnerable devices
```

Reply:
I caught this in my current pass with the automated script. Much more first pass reliable than copy pasta